scene/3d/skeleton_3d.cpp

```diff
@@ -265,11 +265,25 @@ void Skeleton3D::_update_process_order() {
 
 	bones_backup.resize(bones.size());
 
+	_update_bone_names();
+
 	process_order_dirty = false;
 
 	emit_signal("bone_list_changed");
 }
 
+void Skeleton3D::_update_bone_names() {
+	String names;
+	for (int i = 0; i < bones.size(); i++) {
+		names += i == 0 ? bones[i].name : "," + bones[i].name;
```

Comment:
```suggestion
		if (i > 0) {
			names += ",";
		}
		names += bones[i].name;
```
I'd argue this is easier to read and should also optimise better

Reply:
This was done for consistency with NodePath class.

- NodePath::get_concatenated_names()
- NodePath::get_concatenated_subnames()

Should they be changed as well?

Reply:
Then I'd suggest to follow the current established style and we can look at those separately instead